src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuid } from 'uuid';
+
+const initialState = {
+  books: [],
```

Comment:
This is good 💯 but try to add some books to your `initialState` to make sure that the books from that state are getting added. 🤓

Reply:
Hello Reviewer, this pull request was for the previous milestone and I accidentally submitted it for the second milestone/exercise. Kindly give me any directions on how I will submit the next exercise/project

Reply:
This pull request was for the [Bookstore: add reducers and actions](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md) and accidentally submitted it for the [Bookstore: use Redux in React components.](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components_v2.md), kindly give me any guidance on how I will submit the next exercise,just reliased it after getting the first review from @didierganthier

Reply:
@Olamarx Noted👌